build.xml

```diff
@@ -728,6 +728,20 @@
     <!--                                                                                    -->
 
     <!-- Internal use -->
+    <target name="gradle-install" description="Build and install (in maven local) the uPortal modules that have been ported to a Gradle-based build">
+        <!--
+         | TEMPORARY MEASURE!
```

Comment:
Not yet.  Probably we will need that target as long as we need the `build.xml` file.  We'll be able to remove that file entirely after Phase II (though maybe not immediately).